server.js

```diff
@@ -94,30 +106,65 @@ app.get('/test', function(req, res) {
   // var params = {id: 2471390}; // Phoenix
   // var params = {id: }; // 
 
-  client.get('trends/place', params, function(error, tweets, response){
-    if (error) {
-      console.log('ERROR OCCURED', error);
-      throw error;
-    }
-    
-    var apiArr = tweets;
-
-    for (var i = 0; i < apiArr[0].trends.length; i++) {
-      new LocationTrend({
-        location: apiArr[0].locations[0].name,
-        trend_name: apiArr[0].trends[i].name,
-        tweet_volume: apiArr[0].trends[i].tweet_volume
-      })
-      .save(function(err) {
-        if(err) throw err;
-        console.log('trend saved!');
-      });
-    }
-
-    res.status(200);
-    res.send(tweets);
-
-  });
+  paramArray = [{
+        id: 2487956
+      },
+      // {id: 2487889},
+      // {id: 2379574},
+      // {id: 2450022},
+      {
+        id: 2459115
+      }, {
+        id: 2514815
+      }, {
+        id: 2490383
+      }, {
+        id: 2436704
+      },
+      // {id: 2475687},
+      {
+        id: 2391279
+      }, {
+        id: 2486340
+      }, {
+        id: 2442047
+      }, {
+        id: 2471390
+      }
+    ]
+    // this for loop was to populate db with all locations listed above, making an api call to each location id
+  for (var i = 0; i < paramArray.length; i++) {
+    var params = paramArray[i];
+
+    client.get('trends/place', params, function(error, tweets, response) {
+      if (error) {
+        console.log('ERROR OCCURED', error);
+        throw error;
+      }
+
+      var apiArr = tweets;
+
+      for (var i = 0; i < apiArr[0].trends.length; i++) {
+        if (apiArr[0].trends[i].tweet_volume) {
+          new LocationTrend({
+              location: apiArr[0].locations[0].name,
+              trend_name: apiArr[0].trends[i].name,
+              tweet_volume: apiArr[0].trends[i].tweet_volume
+            })
+            .save(function(err) {
+              if (err) {
+                throw err;
+              }
+              console.log('trend saved to db!');
+            });
+        }
+      }
+      // res.status(200); // need to comment these out if you want to loop through several api calls to populate db
+      // res.send(tweets); // 
+
+    });
+
+  } // end for loop bracket
 });
 
```

Comment:
Do you think lines 109-167 could go in their own separate file since they are just calling mongo and not acting as a server? This might be a good candidate for a refactor into the utilities folder under twitterAPI.